src/Models/Context/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -39,6 +44,9 @@ export const DeeplinkContextReducer: (
         switch (action.type) {
             case DeeplinkContextActionType.SET_ADT_URL: {
                 draft.adtUrl = action.payload.url || '';
+                if (action.payload.url) {
```

Comment:
why do we care whether there is value? If I call this with null wouldn't we want it to clear the value in local storage? It'd be weird to have it persisted.

Reply:
Fixed: https://github.com/microsoft/iot-cardboard-js/pull/684/commits/5a2a6ad1e8181718ff0f6fa4143c38f6ad4e383d